client/webpack.config.js

```diff
@@ -226,6 +227,7 @@ const webpackConfig = {
 				cacheIdentifier,
 				cacheCompression: false,
 				exclude: /node_modules\//,
+				plugins: [ isDevelopment && require.resolve( 'react-refresh/babel' ) ].filter( Boolean ),
```

Comment:
nit: could be also
```js
plugins: isDevelopment ? [ require.resolve( 'react-refresh/babel' ) ] : []
```
which IMO is more elegant.

Reply:
Fixed in dfd3ee941e